package.json

```diff
@@ -35,23 +35,23 @@
     "cognito"
   ],
   "peerDependencies": {
-    "@angular/common": "^4.3.3",
-    "@angular/core": "^4.3.3",
-    "aws-sdk": "^2.94.0",
-    "rxjs": "^5.4.2"
+    "@angular/common": "^5.1.3",
+    "@angular/core": "^5.1.3",
+    "aws-sdk": "^5.5.6",
```

Comment:
Version is probably a copy-paste from the RxJS ones?

Reply:
I've used the @latest version of these libraries.

Reply:
5.5.6 is not a valid AWS sdk version. https://www.npmjs.com/package/aws-sdk